BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Telemetry.cs

```diff
@@ -32,12 +32,11 @@ public static string WriteTelemetryName(this ITelemetry telemetry, string teleme
         {
             // A different event name prefix is sent for normal mode and developer mode.
             // Format the event name using the following format:
-            // Microsoft.ApplicationInsights[.Dev].<normalized-instrumentation-key>.<event-type>
```

Comment:
Are we certain that no existing experiences depend on the envelope name being formatted as "Microsoft.ApplicationInsights[.Dev].IKey.EventName", e.g. VS integration or anything in the pipeline?

Reply:
I think whole business of envelope names was for the internal customers ingesting data via Vortex. I'd suggest to eliminate this all together and simply use EnvelopeName. If anybody wants to do string formatting - they can write a telemetry initializer

Reply:
this prefix as well as ikey was introduced for internal Vortex customers. Let's remove it as they will need to create telemetry initializer anyway. So they can take care of it in that initializer